expenses-manager-backend/src/main/java/com/endava/expensesmanager/config/WebConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.endava.expensesmanager.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class WebConfig implements WebMvcConfigurer {
+
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/**");
```

Comment:
From documentation "By default, all origins and GET, HEAD, and POST methods are allowed.". 

We need to allow GET, POST, PATCH, PUT, DELETE and OPTIONS